src/Cake.Common/Tools/InspectCode/InspectCodeSettings.cs

```diff
@@ -114,5 +114,11 @@ public sealed class InspectCodeSettings : ToolSettings
         /// Gets or sets the minimal severity of issues to report.
         /// </summary>
         public InspectCodeSeverity? Severity { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to skip analysis of the file
+        /// that was output by the command line tool or not.
+        /// </summary>
+        public bool DoNotAnalyseOutput { get; set; }
```

Comment:
```suggestion
        public bool SkipOutputAnalysis { get; set; }
```

Reply:
Naming things is hard... but that is way better. :)
I will rename it, when I am done working for today. ;)

Reply:
Excellent 👍, please rebase against the latest develop too.